wazo_dird/plugins/wazo_user_backend/api.yml

```diff
@@ -149,6 +149,9 @@ paths:
 definitions:
   WazoContact:
     properties:
+      id:
+        type: number
```

Comment:
I think the `number` type is usually used for `double` and `float` types.

Reply:
you're right